lib/cn/cn_tree.c

```diff
@@ -1674,55 +1727,62 @@ cn_comp_commit(struct cn_compaction_work *w)
             km.km_nodeid = w->cw_node->tn_nodeid;
         }
 
+
         /* CNDB: Log kvset add records.
          */
-        w->cw_err = cndb_record_kvset_add(
-                        w->cw_tree->cndb, w->cw_cndb_txn, w->cw_tree->cnid,
+        err = cndb_record_kvset_add(
```

Comment:
I see a mix of err and w->cw_err usage in this function, any reason for that?

Reply:
I think I missed this when rebasing. I'll change it all to one.